lib/battle/CBattleInfoEssentials.cpp

```diff
@@ -185,6 +185,21 @@ const CStack* CBattleInfoEssentials::battleGetStackByID(int ID, bool onlyAlive)
 		return stacks[0];
 }
 
+const CStack* CBattleInfoEssentials::battleGetStackByUnitId(int unitId, bool onlyAlive) const

```

Comment:
Please remove. This method is identical to `CBattleInfoEssentials::battleGetStackByID`.
Yes, it calls `unitId()` instead of accessing ID, but if you look up `CStack::unitId()` you'll see that this method simply returns ID